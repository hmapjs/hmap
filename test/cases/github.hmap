html
  head
    title #{personal-title}
body
  li.vcard-detail.pt-1.css-truncate.css-truncate-target
    a(href=link)
