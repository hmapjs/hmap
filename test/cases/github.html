<html>
<head>
  <title>moonrailgun</title>
</head>
<body>
  <div>
    aaa
  </div>
  <div>
    vbb
  </div>
</body>
</html>
