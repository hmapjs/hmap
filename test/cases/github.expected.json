{
  "type": "Block",
  "nodes": [{
    "type": "Tag",
    "name": "html",
    "selfClosing": false,
    "block": {
      "type": "Block",
      "nodes": [{
        "type": "Tag",
        "name": "head",
        "selfClosing": false,
        "block": {
          "type": "Block",
          "nodes": [{
            "type": "Tag",
            "name": "title",
            "selfClosing": false,
            "block": {
              "type": "Block",
              "nodes": [{
                "type": "Code",
                "val": "personal-title",
                "buffer": true,
                "mustEscape": true,
                "isInline": true,
                "line": 3,
                "filename": "github.hmap"
              }],
              "line": 3,
              "filename": "github.hmap"
            },
            "attrs": [],
            "attributeBlocks": [],
            "isInline": false,
            "line": 3,
            "filename": "github.hmap"
          }],
          "line": 2,
          "filename": "github.hmap"
        },
        "attrs": [],
        "attributeBlocks": [],
        "isInline": false,
        "line": 2,
        "filename": "github.hmap"
      }],
      "line": 1,
      "filename": "github.hmap"
    },
    "attrs": [],
    "attributeBlocks": [],
    "isInline": false,
    "line": 1,
    "filename": "github.hmap"
  }, {
    "type": "Tag",
    "name": "body",
    "selfClosing": false,
    "block": {
      "type": "Block",
      "nodes": [{
        "type": "Tag",
        "name": "li",
        "selfClosing": false,
        "block": {
          "type": "Block",
          "nodes": [{
            "type": "Tag",
            "name": "a",
            "selfClosing": false,
            "block": {
              "type": "Block",
              "nodes": [],
              "line": 6,
              "filename": "github.hmap"
            },
            "attrs": [{
              "name": "href",
              "val": "link",
              "mustEscape": true
            }],
            "attributeBlocks": [],
            "isInline": true,
            "line": 6,
            "filename": "github.hmap"
          }],
          "line": 5,
          "filename": "github.hmap"
        },
        "attrs": [{
          "name": "class",
          "val": "\"vcard-detail pt-1 css-truncate css-truncate-target\"",
          "mustEscape": true
        }],
        "attributeBlocks": [],
        "isInline": false,
        "line": 5,
        "filename": "github.hmap"
      }],
      "line": 4,
      "filename": "github.hmap"
    },
    "attrs": [],
    "attributeBlocks": [],
    "isInline": false,
    "line": 4,
    "filename": "github.hmap"
  }],
  "line": 0,
  "filename": "github.hmap"
}
